net/MapViewController.m

```diff
@@ -316,16 +325,23 @@ -(void)mapView:(GMSMapView *)mapView didTapAtCoordinate:(CLLocationCoordinate2D)
 #pragma mark Tab Bar
 -(void)mapView:(GMSMapView *)mapView didLongPressAtCoordinate:(CLLocationCoordinate2D)coordinate
 {
-    self.mapView.selectedMarker = nil;
-    [[self navigationController] setNavigationBarHidden:YES animated:NO];
-    [self requestGoogleApiPlace:coordinate];
-    [self requestCategories];
-    [self addBorderColor];
-    [UIView animateWithDuration:0.5 animations:^(void){
-        self.scrollViewLeadingConstraint.constant = 0;
-        [self hideTabBar];
-        [self.view layoutIfNeeded];
-    }];
+    // Only logged one can add new issue
+    if (self.currentUser != nil){
+        self.mapView.selectedMarker = nil;
+        [self requestGoogleApiPlace:coordinate];
+        [self requestCategories];
+        [self addBorderColor];
+        [self customiseProgressBarView];
```

Comment:
do you really need to call 
[self requestCategories];
[self addBorderColor];
[self customiseProgressBarView];
every time user makes long tap?
:)